projects/ccf-rui/src/app/modules/review/review-button/review-button.component.html

```diff
@@ -1,3 +1,3 @@
 <button [class.disabled]="disabled" mat-button  class="review-button" (click)="registerButtonClick($event)" [matTooltip]="disabled ? 'Required: Select an organ and position tissue block inside organ.' : 'Review registration and submit/download.'">
-    {{ embeddedMode ? 'REVIEW AND REGISTER' : 'REVIEW AND DOWNLOAD' }}
+    {{ cancelRegistrationCallbackSet ? 'REVIEW AND REGISTER' : 'REVIEW AND DOWNLOAD' }}
```

Comment:
This should actually be based on when registration callback is set (ie they have a function that will do the registration instead of our default downloading).